README.md

```diff
@@ -22,31 +22,32 @@
 
 ## Why PyGMT?
 
-A beautiful map is worth a thousand words. To truly understand how powerful PyGMT is, play with it online on
-[Binder](https://github.com/GenericMappingTools/try-gmt)! For a quicker introduction, check out our
-[3 minute overview](https://youtu.be/4iPnITXrxVU)!
+A beautiful map is worth a thousand words. To truly understand how powerful PyGMT is,
+play with it online on [Binder](https://github.com/GenericMappingTools/try-gmt)! For a
+quicker introduction, check out our [3 minute overview](https://youtu.be/4iPnITXrxVU)!
 
-Afterwards, feel free to look at our [Tutorials](https://www.pygmt.org/latest/tutorials), visit the
-[Gallery](https://www.pygmt.org/latest/gallery), and check out some
+Afterwards, feel free to look at our [Tutorials](https://www.pygmt.org/latest/tutorials),
+visit the [Gallery](https://www.pygmt.org/latest/gallery), and check out some
 [external PyGMT examples](https://www.pygmt.org/latest/external_resources.html)!
 
-![Quick Introduction to PyGMT YouTube Video](doc/_static/scipy2022-youtube-thumbnail.jpg)
+![Quick Introduction to PyGMT YouTube Video](https://raw.githubusercontent.com/GenericMappingTools/pygmt/refs/heads/main/doc/_static/scipy2022-youtube-thumbnail.jpg)
```

Comment:
Changing `doc/_static/scipy2022-youtube-thumbnail.jpg` to 
`https://raw.githubusercontent.com/GenericMappingTools/pygmt/refs/heads/main/doc/_static/scipy2022-youtube-thumbnail.jpg`.